test/.eslintrc.json

```diff
@@ -1,13 +1,12 @@
 {
+	"root": true,
+
+	"extends": "../.eslintrc-browser.json",
+
 	"env": {
 		"browser": true
```

Comment:
(Although it seems `.eslintrc-browser.json` should have a comment explaining that)